tests/terraform/checks/resource/azure/test_AzureFrontDoorEnablesWAF.py

```diff
@@ -111,6 +111,55 @@ def test_success(self):
         scan_result = check.scan_resource_conf(conf=resource_conf)
         self.assertEqual(CheckResult.PASSED, scan_result)
 
+        def test_success(self):
```

Comment:
it would be great, if you can migrate the tests here to the better syntax with a separate TF file, like one of James PR #2730

Reply:
Updated. Let me know if this works.